Source/Core/VideoCommon/GeometryShaderManager.cpp

```diff
@@ -35,12 +37,9 @@ void GeometryShaderManager::Shutdown()
 
 void GeometryShaderManager::Dirty()
 {
-	SetViewportChanged();
-	SetProjectionChanged();
-	SetLinePtWidthChanged();
-
-	for (int i = 0; i < 8; i++)
-		SetTexCoordChanged(i);
```

Comment:
They all still have to be called on initialization. Maybe remove the Dirty() call in DoState?